### DISCUSSION TITLE
Distinguishable entries added by scheduled items

### DISCUSSION THREAD

**Discussion by skr3ch** [mtotschnig/MyExpenses#1505] [Ideas]
The app allows to create a model and scheduling it.
A feature of models it the button "Apply" on the push to allow auto-adding expense. Could be useful to include an appearance flag on the entry just to know quickly that becomes from a scheduled model.

**Comment by mtotschnig** [comment]
@skr3ch I am not sure I understand your point. The app currently distinguishes models and plans by different icons.
![grafik](https://github.com/mtotschnig/MyExpenses/assets/1485456/40d05f04-e9ee-416c-ba7f-848c7dc37684)
Could you explain what exactly you would like to change?

**Reply by skr3ch** [reply]
I was talking about the expenses list. But I thinked also to a workaround.
Because if while creating a scheduled model you apply a dedicated tag, it will can be distinguished in the list.
Thank you anyway

**Reply by mtotschnig** [reply]
@skr3ch Thanks for the explanation. Sorry for the late reaction. I get your point now. When you open the details for a transaction, we already provide this information, but representing it via an icon in the list is a very good idea. I'll take this up shortly.
